### DISCUSSION TITLE
Question about embB p.Pro371Arg in Resistance Report between v6.2.1 vs v6.3.0

### DISCUSSION THREAD

**Discussion by taranewman** [jodyphelan/TBProfiler#385] [Q&A]
Hi Jody,

Was there an update to the logic about which mutations are output in the Resistance section of the report between v6.2.1 and v6.3.0?

When running tbprofiler on the same isolate, I noticed embB p.Pro371Arg was output in the "Resistance variants report" section in v6.2.1, but output under the "Other variants report" in v6.3.0. Are all mutations with a confidence of "Uncertain significance" now no longer output in the Resistance variants report?

Thanks for your time,
Tara

**Comment by taranewman** [comment]
Please disregard this question, thanks! :) https://github.com/jodyphelan/tbdb/commit/a5e1d487810ac568f354191351f2de7086546c78

**Reply by jodyphelan** [reply]
Glad you found the answer! :)
